lib/web/apiserver.go

```diff
@@ -352,6 +352,9 @@ func NewHandler(cfg Config, opts ...HandlerOption) (*RewritingHandler, error) {
 	h.POST("/webapi/trustedcluster", h.WithAuth(h.upsertTrustedClusterHandle))
 	h.DELETE("/webapi/trustedcluster/:name", h.WithAuth(h.deleteTrustedCluster))
 
+	h.GET("/webapi/apps/:fqdn", h.WithAuth(h.getAppFQDN))
+	h.GET("/webapi/apps/:fqdn/:clusterName/:publicAddr", h.WithAuth(h.getAppFQDN))
```

Comment:
where are either of these used?

Reply:
Nowhere yet. It's going to be used by the [`webapps` app launcher](https://github.com/gravitational/webapps/blob/master/packages/teleport/src/AppLauncher/useAppLauncher.ts). I've got a rough patch prepared against that repo.